spec/.eslintrc

```diff
@@ -0,0 +1,26 @@
+{
```

Comment:
`.eslintrc` is deprecated, use `.eslintrc.json` - http://eslint.org/docs/user-guide/configuring#configuration-file-formats